src/CMakeLists.txt

```diff
@@ -541,6 +543,7 @@ set(TEST_HEADERS
     test/unit/algorithms/ConnectionsTest.hpp
     test/unit/algorithms/NearestNeighborUnitTest.hpp
     test/unit/algorithms/SpatialPoolerTest.hpp
+    test/unit/algorithms/TemporalMemoryTest.hpp
     test/unit/engine/InputTest.hpp
```

Comment:
Python TM unit test has been left pointing to the Python, and this is the port to check the C++ side.